invenio_theme/assets/semantic-ui/less/invenio_theme/theme/modules/checkbox.overrides

```diff
@@ -39,3 +39,50 @@
 .dash:before  { content: '\e801'; }
 .plus:before { content: '\e802'; }
 */
+
+.ui.toggle.checkbox.on-off {
+  position: relative;
+
+  label {
+    position: relative;
+    height: @onOffToggleHeight;
+    padding-left: @onOffToggleWidth;
+
+    &::before {
+      content: "OFF";
```

Comment:
unfortunately we won't be able to translate this, it will never show up in the translation messages

Reply:
I know, but I don't think it should be translated either, as it might change the layout. However, if it's an issue I guess we can move this to ZenodoRDM.